### PR TITLE
Repo store/access proposal

### PR DESCRIPTION
**Pull request by laurentiuNiculae**
**What type of PR is this?**
documentation

**What does this PR do / Why do we need it**:
This PR presents the repository store access proposal where we discuss about a way to store/access easily and efficiently information about repositories and images(repo:tag).

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by andaaron — DISMISSED** [review]
I would also include the sources used to build the images in case they need to be updated.

**Comment by rchincha** [comment]
This should probably land in docs-zot rather than here then.

File: docs/repository-info-store-access/repository_store_access.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+
+type RepoMetadata struct {
+    Tags         map[string]string
+    Signatures   []string
+    Stars        int
+    Description  string
+    LogoPath     string
+}
+
+type SignatureMetadata struct {
```
Comment (shimish2) [inline docs/repository-info-store-access/repository_store_access.md:257]: Empty struct ?

**Comment by andaaron** [comment]
Let's close here. It's been already implemented and the docs are out of date.

**Pull request closed without merging**